### PR TITLE
Bugfixed/dev tools recommendations

### PR DESCRIPTION
**Pull request by pablomarga**
Hi team this PR closes [#2675](https://github.com/wazuh/wazuh-kibana-app/issues/2675). There was a syntaxis error that suggested the user to write `?&` which is a wrong query: 
image
This syntax error that was showed is fixed. Now it removes the `?` when it is wrong and if you got more than one `?` no hints are showed :
Grabación-de-pantalla-2020-12-15-a-las-10 55 40

### CODE REVIEW

**Comment by pablomarga** [comment]
I have also modified the CHANGELOG due to there was a PR related to revision 4015 that was written in revision 4014 paragraph, concretely this one :`- Empty inventory data in Solaris agents [#2680](https://github.com/wazuh/wazuh-kibana-app/pull/2680)
`

**Comment by frankeros** [comment]
Hi @pablomarga 
Testing this branch I found a bug 
when the first suggestions are shown it is adding an empty option
image

It should show no empty options
image

File: public/controllers/dev-tools/dev-tools.js [hunk 1]
```diff
@@ -367,7 +367,7 @@ export class DevToolsController {
     const currentState = AppState.getCurrentDevTools();
     if (!currentState) {
       const demoStr =
-        'GET /agents?status=active\n\n#Example comment\nGET /manager/info\n\nGET /syscollector/000/packages?search=ssh&limit=1\n\nPOST /agents\n' +
+        'GET /agents?status=active\n\n#Example comment\n\n#You can use ? if you need hints \n#for your query\n\nGET /manager/info\n\nGET /syscollector/000/packages?search=ssh&limit=1\n\nPOST /agents\n' +
```
Comment (frankeros) [inline public/controllers/dev-tools/dev-tools.js:370]: I don't understand what this message means. 
does It mean that if I add `?` in any place the suggestions will be shown? cause this happens  only when I add the first `?`

**Review by frankeros — APPROVED** [review]
LGTM! Great job!

**Review by Desvelao — APPROVED** [review]
LGTM! Good job!

**Review by gabiwassan — APPROVED** [review]
👍🏼

**Pull request merged**